modules/account-lib/src/coin/trx/wrappedBuilder.ts

```diff
@@ -0,0 +1,117 @@
+import BigNumber from 'bignumber.js';
+import { BaseCoin as CoinConfig } from '@bitgo/statics';
+import { BaseKey } from '../baseCoin/iface';
+import { BaseTransaction } from '../baseCoin';
+import { InvalidTransactionError } from '../baseCoin/errors';
+import { Transaction } from './transaction';
+import { Address } from './address';
+import { TransactionBuilder } from './transactionBuilder';
+import { KeyPair } from './keyPair';
+import { decodeTransaction } from './utils';
+import { ContractType } from './enum';
+import { ContractCallBuilder } from './contractCallBuilder';
+
+/**
+ * Wrapped Builder class
+ * This builder is created to maintain compatibility with the current uses of account-lib
+ * It has an instance of Transaction Builder or Contract Call Builder as required.
+ */
+export class WrappedBuilder extends TransactionBuilder {
+  private _builder: TransactionBuilder;
+  // private _builder: TransactionBuilder | ContractCallBuilder;
```

Comment:
is this for the future? cleanup dead code please

Reply:
I've removed it.

Reply:
I've removed it.